packages/api/lib/granule-delete.js

```diff
@@ -64,7 +67,9 @@ const deleteGranuleAndFiles = async ({
         }
       );
 
-      await granulePgModel.delete(trx, { granule_id: pgGranule.granule_id });
+      // TODO: relying on the cumulus_id from the lookup is icky, but we need to
```

Comment:
Looking at the code, everywhere we call it we're pulling in a dynamoGranule.collectionId *or* not passing one in, it's probably not the worst thing to expand this method's call signature.     That said, I'm fine sorting that in Another PR(tm).